go/engine/saltpack_encrypt_test.go

```diff
@@ -49,6 +51,72 @@ func TestSaltpackEncrypt(t *testing.T) {
 	run([]string{u1.Username, u2.Username, u3.Username})
 }
 
+type receiverKeys struct {
+	_struct       bool   `codec:",toarray"`
+	ReceiverKID   []byte `codec:"receiver_key_id"`
+	PayloadKeyBox []byte `codec:"payloadkey"`
+}
+
+type encryptionHeader struct {
```

Comment:
I think it makes sense to export these fields from `saltpack` if you need them here.